### PR TITLE
Feature/HomeView/NavBar

### PR DESCRIPTION
**Pull request by LizOrtizR**
Fix styles/Add new components for HomePage
In this commit:
-fix the Page component. modifying and adding styles to the button and arrange the size of the principal image -Added RiseTalk Logo to the NavBar component
-Created a Bloglist Component for the Home Page section.
-Create a CardsBlog Component  for the Home page section.
-Modification  in Layout routes
-Add mages  for use in components

This commit introduces new components to enhance the homepage  and improve user experience. The added components will contribute to a more visually appealing and functional website design.

HomePage View/ Navbar Component result 
Captura de Pantalla 2023-05-10 a la(s) 17 58 56

Reading BlogList  Component  
Captura de Pantalla 2023-05-10 a la(s) 17 59 32

Related Blog Component 
Captura de Pantalla 2023-05-10 a la(s) 19 22 55

Related Blog Component;

### CODE REVIEW

File: app/page.js [hunk 1]
```diff
@@ -1,112 +1,47 @@
...
+            <div className="h-1 w-20 rounded-full mt-8"></div>
+         
+     
+            <p className="text-lg font-medium text-gray-700">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempos Lorem ipsum dolor sitamet, consectetur adipiscing elit, sed do eiusmod tempor 
+            </p>
+            
+            
+            <div className="mt-10 flex">
+              <Link className="px-6 py-2 text-gray-900 font-poppins" href="/courses">Start learning Now</Link>
```
Comment (Jessemccallums) [inline app/page.js:27]: Task remaining: Add orange style to the Link Button,

This is an Example of it, from figma

image

File: app/page.js [hunk 2]
```diff
@@ -1,112 +1,47 @@
...
+              <div className="ml-6">
+               
+              </div>
+            </div>
+          </div>
         </div>
-      </div>
+        <div className="m-20 flex aling">
+        <Image className= "object-right-top "src="/Group40.png" alt="Example Image" width={779} height={526.88} />
```
Comment (Jessemccallums) [inline app/page.js:35]: Improve accuracy of the image with the inspiration of the figma picture design. 

image

File: app/page.js (hunk 1 above)
Comment (LizOrtizR) [reply app/page.js]: Added Orange style to the Link Button 
Captura de Pantalla 2023-05-10 a la(s) 19 07 52

File: app/page.js (hunk 2 above)
Comment (LizOrtizR) [reply app/page.js]: Style image modified 
Captura de Pantalla 2023-05-10 a la(s) 17 58 56

File: public/Group271.png
Comment (LizOrtizR) [inline public/Group271.png:1]: Reading Blog list Result. 
Captura de Pantalla 2023-05-10 a la(s) 17 59 32

**Pull request merged**